GoogleForCreators/web-stories-wp
# Shopping: display product pill in editor

## Context



Right now in the editor when you have a bunch of products on the canvas it's difficult to know which is which.

## Summary



To remedy this, this PR adds a product price tooltip on hover, just like we do for links. The difference is that the tooltip is styled more like the product pill that you see on the frontend when viewing the story.

## Relevant Technical Choices



## To-do



## User-facing changes



![Screenshot 2022-06-28 at 20 27 37](https://user-images.githubusercontent.com/841956/176410642-b7068135-f8ac-4797-9893-4367638f763a.png)


## Testing Instructions






This PR can be tested by following these steps:

1. Add product(s) to page
2. Unselect product element
3. Verify that product price is shown on hover, in a similar style to what you see when you preview the story on the frontend
4. Verify the tooltip is not shown when moving around the element


## Reviews

### Does this PR have a security-related impact?



No

### Does this PR change what data or activity we track or use?



No

### Does this PR have a legal-related impact?



No

## Checklist




---



Fixes #
